lang/en/local_smartmedia.php

```diff
@@ -45,14 +45,21 @@
 $string['provision:sqsqueue'] = 'SQS queue URL: {$a}';
 $string['provision:stackcreated'] = 'Cloudformation stack created. Stack ID is: {$a}';
 $string['provision:uploadlambdaarchives'] = 'Uploading Lambda function archives to resource S3 bucket';
+$string['dashboard:heading'] = 'Smart Media Dashboard';
 $string['report:duration'] = 'Duration (sec)';
 $string['report:format'] = 'Format';
-$string['report:heading'] = 'Smart Media Report ({$a})';
+$string['report:heading'] = 'Smart Media Overview Report ({$a})';
 $string['report:nocostdata'] = ' - ';
 $string['report:resolution'] = 'Resolution';
 $string['report:size'] = 'Size (Mb)';
 $string['report:summary:filesummary'] = 'All file record summary';
+$string['report:summary:processsummary'] = 'Multimedia file process summary';
```

Comment:
Spelling in 'processsummary'

Reply:
What am I missing?

Reply:
Haha no it's me, you're right, the three 's' put me off, ignore this